File: LeetCode/1637两点之间不包含任何点的最宽垂直面积.cpp
int cmp(const void *a, const void *b) {
    return *(int *)a - *(int *)b;
}

int maxWidthOfVerticalArea(int** points, int pointsSize, int* pointsColSize){
    int num[pointsSize];
    memset(num, 0, sizeof(num));
    for (int i = 0; i < pointsSize; i++) {
        num[i] = points[i][0];
    }
    qsort(num, pointsSize, sizeof(int), cmp);
    int max = INT_MIN;
    for (int i = 1; i < pointsSize; i++) {
        max = fmax(max, num[i] - num[i - 1]);
    }
    return max;
}

File: LeetCode/1637两点之间不包含任何点的最宽垂直面积.go
func maxWidthOfVerticalArea(points [][]int) int {
    sort.Slice(points, func(i, j int) bool {
        return points[i][0] < points[j][0]
    })
    ans := 0
    for i := 1; i < len(points); i++ {
        ans = int(math.Max(float64(ans), float64(points[i][0] - points[i - 1][0])))
    }
    return ans
}